test/core/security/grpc_crl_provider_test.cc

```diff
@@ -44,35 +40,11 @@ using experimental::CertificateInfo;
 using experimental::Crl;
 using experimental::CrlProvider;
 
-namespace {
-
-class TestCrlProvider : public experimental::CrlProvider {
- public:
-  std::shared_ptr<Crl> GetCrl(const CertificateInfo& cert) { return test_crl_; }
-  void SetCrl(absl::string_view crl_string) {
-    absl::StatusOr<std::shared_ptr<Crl>> result = Crl::Parse(crl_string);
-    if (result.ok()) {
-      test_crl_ = *result;
-    } else {
-      test_crl_ = nullptr;
-    }
-  }
-
- private:
-  std::shared_ptr<Crl> test_crl_;
-};
-
-class CrlProviderTest : public ::testing::Test {};
-
-}  // namespace
-
-TEST_F(CrlProviderTest, CrlProviderCanReadCrl) {
+TEST(CrlProviderTest, CanParseCrl) {
```

Comment:
Please also test a case where the parsing fails.

Reply:
Added